src/ripple/app/misc/Manifest.h

```diff
@@ -371,11 +372,11 @@ class ManifestCache
     void
     for_each_manifest(PreFun&& pf, EachFun&& f) const
     {
-        std::lock_guard<std::mutex> lock{read_mutex_};
+        std::lock_guard lock{read_mutex_};
         pf(map_.size ());
-        for (auto const& m : map_)
+        for (auto const& [_, manifest] : map_)
         {
-            f(m.second);
+            f(manifest);
```

Comment:
Do we need to explicitly say `(void) _` here?

Reply:
Apparently gcc and clang aren't warning about this, so no `(void)_` needed, and I'll remove the ones I already added.

Reply:
Fixed in 71c328bbc `* [fold] Remove "maybe unused" warning work around:`